app/javascript/src/components/Dashboard/Notes/NoteList.jsx

```diff
@@ -3,41 +3,48 @@ import React from "react";
 import { Clock, MenuVertical } from "@bigbinary/neeto-icons";
 import { Typography, Tag, Dropdown, Avatar } from "neetoui";
 
-const NotesList = ({ data }) => (
-  <div className="flex w-full flex-col space-y-4">
-    {data.map(note => (
-      <div className="rounded-lg border bg-white p-4 shadow-sm" key={note.id}>
-        <div className="flex justify-between">
-          <Typography component="h2" lineHeight="relaxed" style="h4">
-            {note.title}
-          </Typography>
-          <Dropdown buttonStyle="text" icon={MenuVertical} label="">
-            <li>Edit</li>
-            <li>Delete</li>
-          </Dropdown>
-        </div>
-        <Typography lineHeight="relaxed" style="body2">
-          {note.description}
-        </Typography>
-        <div className="mt-3 flex justify-between border-t pt-4">
-          <Tag label={note.tag} size="small" type="solid" />
-          <div className="flex items-center space-x-2">
-            <Clock color="#68737D" size={12} />
-            <Typography style="body3">
-              {note.status} {note.createdAt}
+const NotesList = ({ data, setShowDeleteAlert, setDeleteNoteTitle }) => {
```

Comment:
- Remove unnecessary namespacing
- Fix the mismatch between the component name and the file name